public/js/auth.js

```diff
@@ -1,11 +1,88 @@
-//When we press the enter button, we trigger the login button
-document.getElementById("password")
-    .addEventListener("keyup", function(event) {
-    event.preventDefault();
-    if (event.keyCode === 13) {
-        document.getElementById("login").click(); // or just login()
+//When we press the enter button on the auth page, we trigger the login button
+if (document.getElementById("password")){
+  document.getElementById("password")
+      .addEventListener("keyup", function(event) {
+      event.preventDefault();
+      if (event.keyCode === 13) {
+          document.getElementById("login").click(); // or just login()
```

Comment:
Like you say in the comment, you can call login(), its more efficient :slightly_smiling_face: